code/cryptography/src/rsa/rsa.cpp

```diff
@@ -0,0 +1,120 @@
+#include <iostream>
+#include <math.h>
+#include <stdlib.h>
+#define ll long long
+
+/*
+    e: encryption key
+    d: decryption key
+    Public Key: n, encryptionkey
+    Private Key: n, decryptionkey (or prime1, prime2, totifun)
+*/
+
+ll prime1, prime2, n, totifun, decryptionkey, encryptionkey, text[100], ciphertext[100];
+std::string plaintext;
+
+int isprime(ll n)
+{
+	if (n == 1)
+		return 0;
+	ll sq = sqrt(n);
+	for (int i = 2; i <= sq; ++i)
+	{
+		if (n % i == 0)
+			return 0;
+	}
+	return 1;
+}
+
+void deter()
+{
+	for (int i = 2; i < totifun; ++i)
+	{
+		if (totifun % i == 0)
+			continue;
+		if (!isprime(i))
+			continue;
+		ll res = 1;
+
+		while (1)
+		{
+			res += totifun;
+			if (res % i == 0)
```

Comment:
Reformat file with clang format and 4 spaces.

Reply:
formatted code using clang(x84)